src/Sarif/BlameHunk.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal class BlameHunk : IBlameHunk
+    {
+        private readonly string name;
+        private readonly string email;
+        private readonly string commitSha;
+        private readonly int lineCount;
+        private readonly int finalStartLineNumber;
+
+        public BlameHunk(string name, string email, string commitSha, int lineCount, int finalStartLineNumber)
+        {
+            this.name = name;
+            this.email = email;
+            this.commitSha = commitSha;
+            this.lineCount = lineCount;
+            this.finalStartLineNumber = finalStartLineNumber;
+        }
+
+        public string Name { get => name; }
```

Comment:
add a new line between each property just like linecount and others below

Reply:
Fixed.